include/libdnf5/rpm/package_sack.hpp

```diff
@@ -28,6 +28,7 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #include "libdnf5/base/base_weak.hpp"
 #include "libdnf5/common/exception.hpp"
 #include "libdnf5/common/weak_ptr.hpp"
+#include "libdnf5/rpm/versionlock_config.hpp"
```

Comment:
Or just `#include "versionlock_config.hpp"` on L27 as others from the same folder.